views/templates/admin/diagnostic/conflicts.tpl

```diff
@@ -44,7 +44,7 @@
               <div class="form-wrapper justify-content-center col-xl-12 mt-3 {if empty($conflicts.data)}d-none{/if}">
                 <ul>
                   {foreach $conflicts.data as $conflict}
-                      <li>{$conflict|escape:'html':'UTF-8'}</li>
+                      <li>{$conflict}</li>
```

Comment:
If you don't add nofilter, the module validation will reject automatically the module
`{$conflict nofiler}`

Reply:
Added